config/cluster-runtimes/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../runtimes
+patchesJson6902:
+  - target:
+      group: serving.kserve.io
+      version: v1alpha1
+      kind: ServingRuntime
+      name: ".*"
+    patch: |-
+      - op: replace
+        path: /kind
+        value: ClusterServingRuntime
```

Comment:
If cluster scope is the default install mode, I'm wondering whether we should invert this - change the SRs in `runtimes` to be CSRs, and have this overlay change them to SRs for the namespace scope install case?

Reply:
Again, due to existing FVTs and unit test depend on these to be SR, I try to minimize the changes. A separate PR should address your concern.